vpr/src/base/vpr_constraints.h

```diff
@@ -33,7 +33,7 @@ class VprConstraints {
     /**
      * @brief Return id of the partition the atom belongs to
      *
-     * If an atom is not in a partition (unconstrained), PartitionId::Invalid() is returned.
+     * If an atom is not in a partition (unconstrained), PartitionId::INVALID() is returned.
```

Comment:
At the top of this file, add a short summary explaining the different classes connected to this one (Partition, PartitionRegion, Region)